experiments/AMIP/components/land/bucket_init.jl

```diff
@@ -159,6 +159,46 @@ function ClimaLSM.initialize_drivers(a::CoupledRadiativeFluxes{FT}, coords) wher
     return (;)
 end
 
+"""
+    temp_anomaly_zonally_asymmetric(coord)
+
+Introduce a zonally asymmetric temperature IC anomaly.
+Currently, the amplicture of the anomaly is 0 so no anomaly is applied.
+"""
+function temp_anomaly_zonally_asymmetric(coord)
+    FT = eltype(coord)
+    radlat = coord.lat / FT(180) * pi
+    anom_ampl = FT(0) # this is zero, no anomaly
+    lat_0 = FT(60) / FT(180) * pi
+    lon_0 = FT(-90) / FT(180) * pi
+    radlon = coord.long / FT(180) * pi
+    stdev = FT(5) / FT(180) * pi
+
+    anom = anom_ampl * exp(-((radlat - lat_0)^2 / 2stdev^2 + (radlon - lon_0)^2 / 2stdev^2))
+    return anom
+end
+
+"""
+    temp_anomaly_aquaplanet(coord)
+
+Introduce a temperature IC anomaly for the aquaplanet case.
+"""
+function temp_anomaly_aquaplanet(coord)
+    FT = eltype(coord)
+    return FT(29) * exp(-coord.lat^2 / (2 * 26^2))
+end
```

Comment:
Document this function, especially because it has some numbers that are not intuitive like 26 and 29. You don't need `FT` here.